dss/stepfunctions/gscopyclient/implementation.py

```diff
@@ -101,3 +104,37 @@ def run(self) -> dict:
         },
     }
 }
+
+
+# Public input/output keys for the copy + write-metadata state function.
+class CopyWriteMetadataKey:
```

Comment:
Should the implementations of the "copy-write-metadata" step functions live in their own directories? Seems to make sense given that the daemon directories themselves are divided up.

Reply:
I would argue not.  The daemon directories are divided up because they have to be, not because I actually want them to be. :)